resources/wikia/modules/cookiePolicy.js

```diff
@@ -0,0 +1,55 @@
+/* global GlobalNotification */
+require(['wikia.cookies', 'wikia.geo'], function (cookies, geo) {
+	'use strict';
+
+	/**
+	 * Initialize JS for whether or not to show a banner notifying European Union users that we use cookies
+	 * on our site, per EU law. Users can dismiss the notification and will not be shown again, unless
+	 * they clear their cookies.
+	 */
+	function initCookieNotification() {
+		if (shouldShowBanner()) {
+			showBanner();
+		}
+	}
+
+	/**
+	 * Check if user is from the EU and hasn't seen the banner yet
+	 * @returns {boolean}
+	 */
+	function shouldShowBanner() {
+		var fromEU = geo.getContinentCode() === 'EU';
+
+		return (fromEU && !hasSeenBanner());
+	}
+
+	/**
+	 * Display the cookie policy message to the user
+	 */
+	function showBanner() {
+		var message = $.msg('oasis-eu-cookie-policy-notification-message') +
```

Comment:
I didn't know about `mw.message`. It looks like it's not used very much yet, but probably should be? It's been an issue for a long time that `$.msg` doesn't do any wikitext parsing, and I actually have an email thread with @macbre and @jcellary about this. I'll see if I can update these strings to use `mw.message` so we can escape them.